.storybook/styles.scss

```diff
@@ -1,4 +1,4 @@
-@use '~@onfido/castor';
+@use '@onfido/castor';
```

Comment:
Did you want to still leave this?

Reply:
Nope, my mistake, thanks.

Reply:
Actually this is the only one Vite builds, I think. It breaks without `~`.
https://vercel.com/onfido-oss/castor/76qkGNNpPMGSP4gsaz5rXUSwc8ft

Reverting.